andela/jiayou-rc
# Feature/#141492899/wallets & funds transfer

**What does this PR do?**
This PR adds the Wallet feature to the project

**Description of Task to be completed?**
Implement Wallets.
Consumer to consumer funds transfer.

**How should this be manually tested?**
On the reaction commerce application, click on the search icon and then type to see real-time matching results.

What are the relevant pivotal tracker stories?
#141492899. Display real-time results as shopper is typing.

Screenshots (if appropriate)
<img width="1452" alt="screen shot 2017-03-24 at 12 53 06 pm" src="https://cloud.githubusercontent.com/assets/24938070/24293390/c7397982-1091-11e7-9185-6278f9e0f70f.png">
